### ISSUE TITLE
train example of syncbn

### ISSUE THREAD

**Issue by boundles** [zhanghang1989/PyTorch-Encoding#21]
Hi, could you provide an example of train syncbn?

**Comment by zhanghang1989** [comment]
This issue has been resolved in favor of releasing PyTorch compatible Synchronized Cross-GPU [encoding.nn.BatchNorm2d](http://hangzh.com/PyTorch-Encoding/syncbn.html) and the [example](https://github.com/zhanghang1989/PyTorch-SyncBatchNorm).
